extensions/sql-database-projects/src/models/publishProfile/publishProfile.ts

```diff
@@ -129,7 +133,169 @@ async function readConnectionString(xmlDoc: any): Promise<{ connectionId: string
 /**
  * saves publish settings to the specified profile file
  */
-export async function savePublishProfile(profilePath: string, databaseName: string, connectionString: string, sqlCommandVariableValues?: Map<string, string>, deploymentOptions?: mssql.DeploymentOptions): Promise<void> {
+export async function savePublishProfile(profilePath: string, databaseName: string, connectionString: string, sqlCommandVariableValues?: Map<string, string>, deploymentOptions?: mssql.DeploymentOptions | vscodeMssql.DeploymentOptions): Promise<void> {
 	const dacFxService = await utils.getDacFxService();
-	await dacFxService.savePublishProfile(profilePath, databaseName, connectionString, sqlCommandVariableValues, deploymentOptions);
+	if (utils.getAzdataApi()) {
+		await (dacFxService as mssql.IDacFxService).savePublishProfile(profilePath, databaseName, connectionString, sqlCommandVariableValues, deploymentOptions as mssql.DeploymentOptions);
+	} else {
+		await (dacFxService as vscodeMssql.IDacFxService).savePublishProfile(profilePath, databaseName, connectionString, sqlCommandVariableValues, deploymentOptions as vscodeMssql.DeploymentOptions);
+	}
+}
+
+export function promptToSaveProfile(project: Project, publishProfileUri?: vscode.Uri) {
+	return vscode.window.showSaveDialog(
+		{
+			defaultUri: publishProfileUri ?? vscode.Uri.file(path.join(project.projectFolderPath, `${project.projectFileName}_1.publish.xml`)),
+			saveLabel: constants.save,
+			filters: {
+				'Publish Settings Files': ['publish.xml'],
```

Comment:
Just want to confirm that this is different on purpose from what SSDT uses, which is "Publish files".  
![image](https://github.com/microsoft/azuredatastudio/assets/31145923/7152d6a8-d92d-4396-a9ad-812a4a77de16)

Reply:
Not intentional, updated it to reflect like SSDT.